PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,434 @@
+#!/bin/bash
+
+# # PreFreeSurferPipeline.sh
+#
+# ## Copyright Notice
+#
+# Copyright (C) 2022-2024 The Human Connectome Project
+#
+# * Washington University in St. Louis
+# * University of Minnesota
+# * Oxford University
+#
+# ## Author(s)
+#
+# * Mikhail Milchenko, Department of Radiology, Washington University in St. Louis
+# * Matthew F. Glasser, Department of Anatomy and Neurobiology, Washington University in St. Louis
+#
+# ## Product
+#
+# [Human Connectome Project][HCP] (HCP) Longitudinal Pipelines
+#
+# ## License
+#
+# See the [LICENSE](https://github.com/Washington-University/Pipelines/blob/master/LICENSE.md) file
+
+
+function create_zero_warp
+{
+    local target_image=$1 warp_file=$2
+
+    # Step 1: Extract dimensions from the target image
+    local dim1=$(fslval $target_image dim1)
+    local dim2=$(fslval $target_image dim2)
+    local dim3=$(fslval $target_image dim3)
+    local pixdim1=$(fslval $target_image pixdim1)
+    local pixdim2=$(fslval $target_image pixdim2)
+    local pixdim3=$(fslval $target_image pixdim3)
+
+    local temp=`mktemp volXXX`.nii.gz
+
+    # Step 2: Create a zero-filled image with the same dimensions
+    fslmaths $target_image -mul 0 $temp
+
+    # Step 3: Merge it into a 4D file with three volumes (all zeros)
+    fslmerge -t $warp_file $temp $temp $temp
+
+    # Optional: Set the voxel dimensions correctly if needed
+    fslcreatehd $dim1 $dim2 $dim3 3 $pixdim1 $pixdim2 $pixdim3 1 0 0 0 16 $warp_file
+}
+
+
+set -eu
+
+# Requirements for this script
+#  installed versions of: FSL
+#  environment: HCPPIPEDIR, FSLDIR
+
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/processingmodecheck.shlib" "$@" # Check processing mode requirements
```

Comment:
T2w files appear to be mandatory in this script, so we don't need any "legacy-data mode" checking, remove this line.

Reply:
They need not be though.  Perhaps we should allow legacy mode longitudinal processing.

Reply:
Right now longitudinal processing depends on PreFreeSurfer where T2w is mandatory (and also run MNI registration script called by PreFreeSurfer where T2w is mandatory). 
So supporting T1w only mandates support of that mode in PreFreeSurfer - do we want to work on that?

Reply:
For now I commented out the processing mode check.

Reply:
I imagine the "T1w only" possibility will be handled in PreFS, so we would never want PostFS to complain about it to the user - we already hassled them about it once.  So, we should probably never do `processingmodecheck.shlib` stuff in any PostFS code.